docs/documentation/helpcenter.md

```diff
@@ -163,6 +169,15 @@ Do not use `we` to refer to Zulip or its creators; for example, "Zulip also
 allows ...", rather than "we also allow ...". On the other hand, `you` is ok
 and used liberally.
 
+### Keyboard shortcuts
+
+Use backticks for each keyboard key in the shortcut (e.g. `Enter` or `R`).
+For shortcuts with more than one key, add a plus (+) surrounded by spaces
+between each key; for example `Ctrl` + `K`, instead of `Ctrl + K` or
+`Ctrl`+`K` or `Ctrl` `K`).
+
+Use non-Mac keyboard keys; for example `Enter` instead of `Return`.
```

Comment:
Let's add a sentence to this paragraph noting that Zulip has JavaScript that will automatically translate Windows keys to the Mac versions for readers with a Mac user agent.

Reply:
I also added information on how to test / check the Mac shortcuts since that seemed good to document here as well.